llvm/test/CodeGen/AMDGPU/addrspacecast.r600.ll

```diff
@@ -42,7 +42,7 @@ define amdgpu_kernel void @addrspacecast_flat_null_to_local(ptr addrspace(1) %ou
 ; CHECK-NEXT:    PAD
 ; CHECK-NEXT:    ALU clause starting at 4:
 ; CHECK-NEXT:     MOV * T0.X, literal.x,
-; CHECK-NEXT:    -1(nan), 0(0.000000e+00)
+; CHECK-NEXT:    0(0.000000e+00), 0(0.000000e+00)
```

Comment:
Does R600 even have multiple address spaces? It doesn't look like the case from the data layout string.